python/dgl/mock_sparse2/sparse_matrix.py

```diff
@@ -173,6 +195,58 @@ def transpose(self):
         """
         return SparseMatrix(self.c_sparse_matrix.transpose())
 
+    def coalesce(self):
```

Comment:
Should mark that this `coalesce` does not handle autograd since it uses torch::sparse_coo_tensor.